mailet/standard/src/test/java/org/apache/james/transport/matchers/HasMimeTypeParameterTest.java

```diff
@@ -45,10 +47,9 @@ void setUp() throws MessagingException {
         sampleMail = FakeMail.builder()
             .mimeMessage(MimeMessageBuilder
                 .mimeMessageBuilder()
-                .addHeader("Content-Type",
-                    "multipart/report; report-type=\"disposition-notification\"; boundary=\"=-ac61K8KXSRpaQ/eveStc\"")
                 .setSubject("Mail read")
-                .setText("You email has been read by Bart"))
+                .addHeader("Content-Type", TEST_CONTENT_TYPE)
+                .setText("You email has been read by Bart", TEST_CONTENT_TYPE))
```

Comment:
why do you need to put the content type here ?

Reply:
else it creates the message with text/plain mimetype

Reply:
maybe we should not call addHeader before, then ?

Reply:
I also tried, but it fails without it. Feel free to debug it :)